presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSinkProvider.java

```diff
@@ -82,7 +82,7 @@ public ConnectorPageSink createPageSink(ConnectorTransactionHandle transactionHa
 
         RaptorInsertTableHandle handle = (RaptorInsertTableHandle) tableHandle;
         return new RaptorPageSink(
-                new FileSystemContext(session),
+                new HdfsContext(session.getIdentity()),
```

Comment:
But schema and table names are not available in RaptorInsertTableHandle.

Reply:
We can ignore schema/table. But we should populate other fields.